### DISCUSSION TITLE
impossible to use jit mode with blocklist

### DISCUSSION THREAD

**Discussion by tcastelly** [tailwindlabs/tailwindcss#7667] [Ideas]
**What version of Tailwind CSS are you using?**

v2.2.19

**What build tool (or framework if it abstracts the build tool) are you using?**

Vite + VueJS + postcss (v8.4.1)

**What version of Node.js are you using?**

v14.17.0

**What browser are you using?**

Chromium

**What operating system are you using?**

Gnu Linux (arch)

**Reproduction URL**

https://github.com/shenron/tailwindcss-jit-blocklist

**Describe your issue**
The jit mode ignore blocklist, because of this it's not possible to avoid duplicate classes with a third tailwindcss party.

**Comment by sntran** [comment]
At least in Tailwind 2.2.x with jit mode, I think only `safelist` option is supported.

**Comment by Farr69** [comment]
Tailwind CSS v2.0 has been updated for the latest PostCSS release which requires installing _postcss_ and _autoprefixer_ as peer dependencies alongside Tailwind itself.
[mcdvoice](https://www.mcdvoice.guru/)

**Comment by adamwathan** [comment]
Hey! Yeah Tailwind doesn't have a blocklist feature and never has, that was a feature of PurgeCSS which we aren't using under the hood anymore in v3. I think a blocklist is a good potential feature idea and have it in my list of things for us to consider though 👍 Going to convert this to a discussion to track as a feature request.

**Comment by LukeTowers** [comment]
@adamwathan this would be very useful to us at @wintercms, especially in a CMS context where multiple plugins can be added arbitrarily by the end user and we'd like to avoid duplicating the classes loaded by the plugins for their own views when they're already included by the core CMS' Tailwind styles.

**Comment by riskihajar** [comment]
i build component for laravel nova which use tailwind as class layout, some class feature not included by compiled css, so i need add some class by custom component and ignore class already exists on nova classes layout, this would be useful
